### ISSUE TITLE
Invalid cassandra-reaper.yml generated when REAPER_JMX_AUTH_PASSWORD contains special chars

### ISSUE THREAD

**Issue by adutra** [thelastpickle/cassandra-reaper#1143]
Reaper version was 2.3.0.

The script `configure-jmx-credentials.sh` does not enclose some env vars in double quotes when printing them to /etc/cassandra-reaper.yml, see [here](https://github.com/thelastpickle/cassandra-reaper/blob/b9c7129361db47da70b56ca589db36929939e891/src/server/src/main/docker/configure-jmx-credentials.sh#L46-L47).

This is a problem, especially when `REAPER_JMX_AUTH_PASSWORD` contains invalid chars:

```
io.dropwizard.configuration.ConfigurationParsingException: /etc/cassandra-reaper.yml has an error:
  * Malformed YAML at line: 99, column: 14; while scanning for the next token; found character %'%' that cannot start any token;  in 'reader', line 98, column 15:
        password: %MpG'+A%n3j9|oG4dXNc
                  ^
 at [Source: (ByteArrayInputStream); line: 98, column: 13]

	at io.dropwizard.configuration.ConfigurationParsingException$Builder.build(ConfigurationParsingException.java:279)
	at io.dropwizard.configuration.BaseConfigurationFactory.build(BaseConfigurationFactory.java:96)
	at io.dropwizard.cli.ConfiguredCommand.parseConfiguration(ConfiguredCommand.java:126)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:74)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:93)
	at io.cassandrareaper.ReaperApplication.main(ReaperApplication.java:109)
Caused by: com.fasterxml.jackson.dataformat.yaml.snakeyaml.error.MarkedYAMLException: while scanning for the next token; found character %'%' that cannot start any token;  in 'reader', line 98, column 15:
        password: %MpG'+A%n3j9|oG4dXNc
```

**Comment by adutra** [comment]
I can provide a PR but @adejanovski what branch should it target? From what I see we could target `2.3`, `3.0` or `master`.

**Comment by adejanovski** [comment]
@adutra, please point it to the 3.0 branch. That'll allow us to release a new patch version with the fix.

**Comment by adutra** [comment]
Done, thanks!

**Comment by adejanovski** [comment]
This was fixed in 3.0.1.
